src/ArchPresets/Base.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pest\ArchPresets;
+
+/**
+ * @internal
+ */
+final class Base extends AbstractPreset
+{
+    /**
+     * Executes the arch preset.
+     */
+    public function execute(): void
+    {
+        $this->expectations[] = expect([
+            'debug_zval_dump',
+            'debug_backtrace',
+            'debug_print_backtrace',
+            'dd',
+            'ddd',
+            'dump',
+            'ray',
+            'die',
+            'goto',  
+            'var_dump',
+            'phpinfo',
+            'echo',
+            'print',
+            'print_r',
+            'var_export',
+            'xdebug_break',
+            'xdebug_call_class',
+            'xdebug_call_file',
+            'xdebug_call_(int',
```

Comment:
```suggestion
            'xdebug_call_int',
```

Reply:
I can't see any proof that that function exists. Was this meant to be `xdebug_call_function` ?

Reply:
@nunomaduro Not sure if you saw this comment